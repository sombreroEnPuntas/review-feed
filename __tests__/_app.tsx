import React from 'react'
import { render } from '@testing-library/react'

import TestedComponent from '../pages/_app'

const TestDiv = (props: React.HTMLProps<HTMLDivElement>) => <div {...props} />

const getDefaultProps = (customProps?: typeof TestedComponent) => ({
  Component: TestDiv,
  pageProps: { children: 'ðŸ‘¾' },
  ...customProps,
})

test('renders Component with passed pageProps', () => {
  const { container } = render(<TestedComponent {...getDefaultProps()} />)

  expect(container).toMatchSnapshot()
})
