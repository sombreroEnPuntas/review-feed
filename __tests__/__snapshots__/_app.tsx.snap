// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Component with passed pageProps 1`] = `
<div>
  <div>
    ðŸ‘¾
  </div>
</div>
`;
